src/__tests__/App.test.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { render, screen, waitFor } from '@testing-library/react';
+import App from "../../src/components/App";
+
+describe("App", () => {
+  test("renders App component correctly", async () => {
+    // jest.spyOn(axios, "get").mockResolvedValue({ data: forecast });
```

Comment:
remove commented out code. Your app doesn't make a `get` request on load, so you do not need to return mock data. Also, this test isn't async so you can remove the `async` key word on line 6.